README.md

```diff
@@ -78,6 +78,9 @@ Below you will find descriptions of all rules available in **AEM Rules for Sonar
 - **AEM-19** Implicit search strategy used in Sling Query
   - `SearchStrategy` can have negative performance impact if mismatched.
   Therefore developer should always make informed decision and define strategy explicitly.
+  
+- **HTL-3** Usage of HTML comments should be avoided if possible
```

Comment:
Move description like in https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/137/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
moved